webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -181,12 +181,40 @@ class RudderCRunner(
     } yield ()
   }
 
+  // Create an empty reporting bundle for compatibility with the fallback process
+  // Can be removed once rudderc is the only generation method
+  def emptyReportingFile(technique : EditorTechnique) = {
+
+    val bundleParams = if (technique.parameters.nonEmpty) technique.parameters.map(_.name.canonify).mkString("(",",",")") else ""
+    val content =
+      s"""bundle agent ${technique.bundleName.value}_rudder_reporting${bundleParams}
+         |{
+         |}"""
+
+    val reportingFile = File(outputPath) / "rudder_reporting.cf"
+
+
+    TechniqueWriterLoggerPure.error(s"PLOUF target '${technique.name}' in path ${reportingFile.path.toString}")
```

Comment:
as that instuction is a pure value, it won't product side effect. If you want (you want) to include it in tha chain of instruction to be interpreted by the `IO` interpreter, you need to bundle it with the following one (which is returned) by adding a `*>` (which means "and pack it to next instruction, I don't care of its returned value"). 
You can also do: 
```
for {
  _ <- TechniqueWriterLoggerPure.error(s"PLOUF target '${technique.name}' in path ${reportingFile.path.toString}")
 _ <- IOResult.effect(s"Could....) { ... }
} yield () 
``` 
which would allows to use intermediate returned values if you needed them.